### PR TITLE
Add SDK table

### PR DESCRIPTION
**Pull request by toddbaert**
- Adds SDK card component
- Adds SDK card table and new page

Direct link to new content in preview: https://deploy-preview-60--lucky-creponne-baf9b3.netlify.app/docs/reference/sdks

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -19,15 +19,6 @@
...
 	"rules":{
-			// ESM imports need extensions, but TS doesn't re-write them.
-			// this forces us to add `.js` to all imports, which makes sure our ESM modules work.
-			"import/extensions": [
-				"error",
-				"ignorePackages",
-				{
-					"js": "always"
-				}
-			],
```
Comment (toddbaert) [inline .eslintrc.json:30]: Don't need this.

File: docs/reference/sdks.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      description: 'OpenFeature SDK for .NET',
+    },
+    {
+      href: 'https://github.com/open-feature/golang-sdk',
+      iconDefinition: faGolang,
+      title: 'Go',
+      description: 'OpenFeature SDK for Golang',
+    },
+  ]}
+/>
```
Comment (toddbaert) [inline docs/reference/sdks.mdx:44]: The cards are rendered based on this table data.

File: package.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
     "@docusaurus/core": "^2.0.1",
     "@docusaurus/preset-classic": "^2.0.1",
     "@fortawesome/fontawesome-svg-core": "^6.1.1",
+    "@fortawesome/free-brands-svg-icons": "^6.1.2",
```
Comment (toddbaert) [inline package.json:24]: This is `CC-BY-4.0 AND MIT`.

File: docs/reference/sdks.mdx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      description: 'OpenFeature SDK for Java',
+    },
+    {
+      href: 'https://github.com/open-feature/node-sdk',
+      iconDefinition: faNodeJs,
+      title: 'Node.JS',
+      description: 'OpenFeature SDK for Node.JS',
+    },
+    {
+      href: 'https://github.com/open-feature/dotnet-sdk',
```
Comment (beeme1mr) [inline docs/reference/sdks.mdx:33]: Add a .NET icon once it's added to Font Awesome

File: docs/reference/sdks.mdx [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+title: SDKs
+---
+
+# SDKs
+
+SDKs for the evaluation of feature flags and the development of OpenFeature compatible components.
+
+## Server SDKs
+
+import { SDKComponentCard } from '@site/src/components/custom';
```
Comment (skyerus) [inline docs/reference/sdks.mdx:13]: It doesn't look like this import is used

Comment (toddbaert) [reply docs/reference/sdks.mdx:13]: I think you're right!

Comment (toddbaert) [reply docs/reference/sdks.mdx]: Fixed. Unfortunately the .mdx extension doesn't have very advanced features - it can't seem to identify unused imports.

**Review by benjiro — APPROVED** [review]

**Pull request merged**